### PR TITLE
SBERDOMA-942 - PDF generation with pagebreaks

### PR DESCRIPTION
**Pull request by sitozzz**
Add helper method for creating infinite PDF document

image

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3643,6 +3643,77 @@
     mkdirp "^1.0.4"
```
Comment (Dimitreee) [inline yarn.lock:3643]: Revert this, please

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/domains/common/utils/pdf.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ const PDF_FORMAT_SETTINGS = {
     // lineSpace - margin right for rest of lines (in css pixels)
     'a4': { pdfWidth: 210, pdfHeight: 297, elementOffset: 10, firstLineOffset: 23, lineSpace: 80 },
     'a5': { pdfWidth: 148, pdfHeight: 210, elementOffset: 10, firstLineOffset: 23, lineSpace: 80 },
+    'fullscreen': { pdfWidth: PDF_REPORT_WIDTH, pdfHeight: 297, elementOffset: 15, firstLineOffset: 23, lineSpace: 80 },
```
Comment (toplenboren) [inline apps/condo/domains/common/utils/pdf.ts:12]: I saw that we named this one `fullscreen` but below width is defined as 1920px. Maybe its worth to rename it to smth like `1080p`?

**Review by toplenboren — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (sitozzz) [reply yarn.lock]: Done

File: apps/condo/domains/common/utils/pdf.ts (hunk 2 above)
Comment (sitozzz) [reply apps/condo/domains/common/utils/pdf.ts]: Fixed

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**